apps/services/user-notification/migrations/20251104103244-add-scope-column.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    // Add scope column as string
+    await queryInterface.addColumn('user_notification', 'scope', {
+      type: Sequelize.STRING,
+      allowNull: true,
+    })
+
+    // Set all existing notifications to have the main document scope
+    await queryInterface.sequelize.query(`
+      UPDATE user_notification
+      SET scope = '@island.is/documents'
+      WHERE scope IS NULL
+    `)
+
+    // Make the column non-nullable after setting values for existing records
+    await queryInterface.changeColumn('user_notification', 'scope', {
+      type: Sequelize.STRING,
+      allowNull: false,
+    })
```

Comment:
This is a risky migration since it doesn't run in a transaction and runs before new service pods are deployed. This can cause 500 errors when saving notifications with old code.

We should try to make all database changes compatible with old code. I.e. by adding this non-nullable column with a default value of @island.is/documents. Later we can update the column and remove the default value.